src/error.cs

```diff
@@ -203,4 +214,32 @@ static bool ShowInternal (Exception e)
 		}
 		return error;
 	}
+
+	public static WarningLevel GetWarningLevel (int code)
```

Comment:
I don't really like copy-pasting code like this, but OTOH I tried sharing more exception-related code once and ended up with [spaghetti code](https://github.com/xamarin/xamarin-macios/blob/7cf9a64656e8a9ae528fe77b475b02df529535a0/src/ObjCRuntime/ErrorHelper.cs#L6-L42). So I guess this is the best of two evils, in particular since this code isn't very complicated (unless you have a better idea :smile:)